save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/demo/management/DemoSettingsComponent.kt

```diff
@@ -16,16 +16,17 @@ import react.dom.html.ReactHTML.input
  *
  * @param demoDto currently configured [DemoDto]
  * @param setDemoDto callback to update [demoDto] state
+ * @param isDisabled flag that defines if input forms are disabled or not
  */
-internal fun ChildrenBuilder.renderDemoSettings(demoDto: DemoDto, setDemoDto: StateSetter<DemoDto>) {
+internal fun ChildrenBuilder.renderDemoSettings(demoDto: DemoDto, setDemoDto: StateSetter<DemoDto>, isDisabled: Boolean) {
```

Comment:
maybe rename to `readOnly` ?

Reply:
We widely use this flag either as `disabled` or `isDisabled` as far as `disabled` is a field of `select`, `input` and other tags

Reply:
Suggestion to leave as is :)